File: System/Spark/Basic/CreatePair/CreatePair.py
from pyspark import SparkConf,SparkContext

conf = SparkConf().setMaster("local").setAppName("My App")
sc = SparkContext(conf = conf)

# Read dataset and create a RDD
inputRDD = sc.textFile("../example.md")

# Create pair-RDD with first word in the sentence as key and the whole sentence as value
pairRDD = inputRDD.map(lambda x :(x.split(" ")[0],x))

# create with function : parallelize()

sc.stop()

File: Framework/Spark/Basic/CreatePair/CreatePair.scala
import org.apache.spark.SparkConf
import org.apache.spark.SparkContext
import org.apache.spark.SparkContext._

val conf = new SparkConf().setMaster("local").setAppName("My App")
val sc = new SparkContext(conf)

// Create RDD
val inputRDD = sc.textFile("../example.md")

// Create pair-RDD
val pairRDD = inputRDD.map(x => (x.split(" ")(0),x))